CHT/Temperature.C

```diff
@@ -84,11 +100,11 @@ bool preciceAdapter::CHT::Temperature::isLocationTypeSupported(const bool meshCo
     // always return true and offload the handling to the user.
     if (meshConnectivity)
     {
-        return true;
+        return (this->locationType_ != LocationType::volumeCenters); // we do not support meshConnectivity for volumeCenters (yet)
```

Comment:
Would a positive formulation ("if faceCenters or faceNodes") be clearer, safer, and consistent, e.g., with the `Pressure.C` and `Velocity.C`?
The comment can remain.